Makefile

```diff
@@ -1,13 +1,21 @@
 VENV=venv
-PYTHON=python
+
 MODULES=qiskit_alice_bob_provider tests
 BUILD_DIR=dist
 BASE_URL=
 API_KEY=
 ifeq ($(OS),Windows_NT)
     ACTIVATE=$(VENV)/Scripts/activate
+    PYTHON3_PATH=$(cmd which python3 2> nul)
 else
     ACTIVATE=$(VENV)/bin/activate
+    PYTHON3_PATH=$(shell which python3 2> /dev/null)
+endif
+
+ifeq ($(PYTHON3_PATH),)
```

Comment:
If the path is null it means there is no Python3 on this machine, hence we will use Python. Does that make sense enough without the comment or should we add one?

Reply:
It's fine like this!